toolbox/fdc3-workbench/src/fixtures/contexts.ts

```diff
@@ -6,6 +6,138 @@ import { ContextItem } from "../store/ContextStore";
 import { v4 as uuidv4 } from "uuid";
 
 export const contexts: ContextItem[] = [
+	{
+		uuid: uuidv4(),
+		id: "Chart example",
+		template: {
+			type: "fdc3.chart",
+			instruments: [
+				{
+					type: "fdc3.instrument",
+					id: {
+						ticker: "AAPL"
+					}
+				},
+				{
+					type: "fdc3.instrument",
+					id: {
+						ticker: "GOOG"
+					}
+				}
+			],
+			range: {
+				type: "fdc3.timeRange",
+				startTime: "2020-09-01T08:00:00.000Z",
+				endTime: "2020-10-31T08:00:00.000Z"
+			},
+			style: "line"
+		},
+		schemaUrl: new URL("https://fdc3.finos.org/schemas/next/context/chart.schema.json"),
+	},
+	{
+		uuid: uuidv4(),
+		id: "Chat Initialization example",
+		template: {
+			type: 'fdc3.chat.initSettings',
+			chatName: 'Chat ABCD',
+			members: {
+				type: 'fdc3.contactList',
+				contacts: [{
+					type: 'fdc3.contact',
+					name: 'Jane Doe',
+					id: {
+						email: 'jane@mail.com'
+					}
+				},{
+					type: 'fdc3.contact',
+					name: 'John Doe',
+					id: {
+						email: 'john@mail.com'
+					},
+				}]
+			},
+			options: {
+				groupRecipients: true, // one chat with both contacts
+				isPublic: false, // private chat room
+				allowHistoryBrowsing: true,
+				allowMessageCopy: true
+			},
+			message: {
+			  type: 'fdc3.message',
+			  text: {
+				'text/plain': 'Hey all, can we discuss the issue together? I attached a screenshot'
+			  },
+			  entities: {
+				 0: {
+					 type: 'fdc3.fileAttachment',
+					  data: {
+					  name: 'myImage.png',
+							dataUri: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAIAQMAAAD+wSzIAAAABlBMVEX///+/v7+jQ3Y5AAAADklEQVQI12P4AIX8EAgALgAD/aNpbtEAAAAASUVORK5CYII'
+					  }
+				  }
+			  }
+			}
+		},
+		schemaUrl: new URL("https://fdc3.finos.org/schemas/next/context/chatInitSettings.schema.json"),
+	},
+	{
+		uuid: uuidv4(),
+		id: "Chat Message example",
+		template:{
+			type: "fdc3.chat.message",
+			chatRoom: {
+				type: 'fdc3.chat.room',
+				providerName: "Symphony",
+				id: {
+					streamId: "j75xqXy25NBOdacUI3FNBH"
+				}
+			},
+			message: "A message to send"
```

Comment:
@kriswest  This the old schema?  Should we merge #1102 and then update this, or will that PR be updated to update this sample?

Reply:
Updated (after #1102 was merged)